angular/projects/admin-nrpti/src/app/records/utils/record-utils.ts

```diff
@@ -381,4 +235,14 @@ export class RecordUtils {
       }
     }
   }
+
+  static getReflectiveInstance(context: any, name: string, ...args: any[]) {
```

Comment:
Does this need to be static? I didn't see it used anywhere else, but could have missed it. Or, maybe it is nice to just have available?

Reply:
Yeah, it doesn't need to be static, but I figured it could get use elsewhere whenver some similar action needs to be done.